### ISSUE TITLE
Activity feed

### ISSUE THREAD

**Issue by johnsillings** [colonistio/design#294]
A lot can happen in Colonist in a short period of time.

As a player, I should be made aware of – and be able to review – changes to game state in the activity feed so I can plan accordingly.

Information about which resources players received, which trades were offered, which trades were accepted, which development cards were played, etc. are important for playing the game strategically.

![Image](https://user-images.githubusercontent.com/33302207/218590027-663eca6b-d051-49fe-87c4-577a2a9070c5.png)

## Steps:

- [x] Look at other turn-based multiplayer strategy games with dice and collect examples in both video & screenshot forms
- [x] Brainstorm and list multiple <rough> designs that meet the acceptance criteria of the task to a satisfactory level. At this stage more exotic designs can be considered as it’s important to widen our view of the solution landscape before focusing too much on details. 
- [x] For each design rate values below from 0 to 5
  - Clarity: How clear is it: https://www.liamcharlton.com/post/signs-feedback-and-the-ux 
  - Ease of use: Number of needed interactions with the UI, complexity of interactions, number of hands needed to complete the actions, thumb zone etc.
  - Acceptance criteria met: 
- [x] Implement changes to update the rating values of the above designs
- [x] Identify any new player issues that may arise from each of your rough designs and the potential for them to be mitigated.
- [x] Decide on 1-2 designs that have the greatest potential to work by analyzing the results from previous steps.
- [x] Share all your results from the previous steps with the team in a table containing concise information; It’s important to also show your thought process for the design decisions you have made.
- [ ] Get the greenlight from project owners to start work on a detailed Figma prototype for one of the high potential rough designs.

## Acceptance criteria:

- [x] Players should be aware of all public changes to the game state (e.g. building improvements, purchasing development cards, offering trades, accepting trades, rolling dice, moving the robber, etc.).
- [x] Players should be able to view historical changes to the game state in an activity feed.
- [ ] The activity feed should not cover or obfuscate important UI elements unless the player is actively interacting with it.
- [x] Related to the above, players should be able to show and hide the activity feed. Design should be sensitive to the limited screen space on mobile.

**Comment by huaniebun** [comment]
Link to research & designs: https://www.figma.com/file/up1U35QOvkl2aCfD0cJXGW/Colonist-Mobile?node-id=752%3A490849&t=sHn7LALTPy4BmqY3-0 

Note: I renamed "activity feed" as "game logs" so as not to be confused with notif activity feed

Shortlisted these options:

### 1. Expandable Menu options upon clicking on menu button + half activity feed, half game board in screen
![Image](https://user-images.githubusercontent.com/98397645/221398483-d0dc3ed6-8dd8-4e88-9b25-5d6418afd342.png)

### 2. Activity Feed Button + Full game log screen
![Image](https://user-images.githubusercontent.com/98397645/221398482-e68edf6f-42fe-4a57-be8f-a05ac33bfd71.png)

### Additional Ideas to explore: Filtering of info in game logs 

A) Filter by actions:
![Image](https://user-images.githubusercontent.com/98397645/221398480-7422a611-5e4a-46f8-b55e-1cc493edf8da.png)

B) Filter by players:
![Image](https://user-images.githubusercontent.com/98397645/221398481-6eacf216-fb27-45fe-b447-9629efa1ecb9.png)

Since activity feed is closely related to chat, I have included how the chat might look like in the Figma file. More details will be worked out in a separate issue for chat.

**Comment by huaniebun** [comment]
For review @demiculus @johnsillings @samgawaran @Ioseffagundes @AlfredTimmer

**Comment by huaniebun** [comment]
Have iterated on the designs and shortlisted these 2 options: https://www.figma.com/file/up1U35QOvkl2aCfD0cJXGW/Colonist-Mobile?node-id=803%3A499790&t=7fsB5CopyBM7rEyp-1 

### Option 1

![Image](https://user-images.githubusercontent.com/98397645/222085626-36a1f99c-bd47-4ca8-9612-68a78b676b7f.png)

### Option 4

![Image](https://user-images.githubusercontent.com/98397645/222085651-7f8b579c-bc39-47b9-9420-0d9b5cdd30a4.png)

**Comment by aalmadan** [comment]
Closed, done

**Comment by JuanRoman77** [comment]
@demiculus should we close this only after an implementation issue is in place?

**Comment by demiculus** [comment]
nah its fine, design task is done, implementation issue will be in dev repo
